pkg/kubelet/kubelet.go

```diff
@@ -1364,6 +1442,10 @@ func (kl *Kubelet) initializeModules() error {
 
 // initializeRuntimeDependentModules will initialize internal modules that require the container runtime to be up.
 func (kl *Kubelet) initializeRuntimeDependentModules() {
+	if err := kl.chownDirForRemappedIDs(kl.getPodsDir()); err != nil {
```

Comment:
It is very conceivable that there are situations when this might not need to happen. For example if [shiftfs ever gets merged](https://lkml.org/lkml/2018/11/2/420) then this approach is disadvantageous.